tests/codemods/test_tempfile_mktemp.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+from codemodder.codemods.tempfile_mktemp import TempfileMktemp
+from tests.codemods.base_codemod_test import BaseSemgrepCodemodTest
+
+
+class TestTempfileMktemp(BaseSemgrepCodemodTest):
+    codemod = TempfileMktemp
+
+    def test_rule_ids(self):
+        assert self.codemod.RULE_IDS == ["secure-tempfile"]
+
+    def test_import(self, tmpdir):
+        input_code = """import tempfile
+
+tempfile.mktemp()
+var = "hello"
+"""
+        expexted_output = """import tempfile
+
+tempfile.mkstemp()
+var = "hello"
+"""
+
+        self.run_and_assert(tmpdir, input_code, expexted_output)
+
+    def test_import_with_arg(self, tmpdir):
+        input_code = """import tempfile
+
+tempfile.mktemp('something')
+var = "hello"
+"""
+        expexted_output = """import tempfile
+
+tempfile.mkstemp('something')
+var = "hello"
+"""
+
+        self.run_and_assert(tmpdir, input_code, expexted_output)
+
+    def test_from_import(self, tmpdir):
+        input_code = """from tempfile import mktemp
+
+mktemp()
+var = "hello"
+"""
+        expexted_output = """import tempfile
+
+tempfile.mkstemp()
+var = "hello"
+"""
+        self.run_and_assert(tmpdir, input_code, expexted_output)
+
+    @pytest.mark.skip()
```

Comment:
I'm gonna guess this breaks the code in this case. If so, you should change the rule/code to ignore calls like this.

Reply:
That's the point of adding skip, so it doesn't run. If I do run this test, nothing breaks, it just gives us the wrong output without the alias (so it's actually valid code, it just removes the aliasing)

Reply:
That's the point of adding skip, so it doesn't run. If I do run this test, nothing breaks, it just gives us the wrong output without the alias (so it's actually valid code, it just removes the aliasing)

Reply:
I prefer skip because it doesn't run the test. Xfail will run it and fail. Tiny time saver. Benefit to xfail if you think we would magically fix this without knowing, which I don't

Reply:
Sorry if I wasn't clear, but I meant if the code produced by this case is correct, as in, it compiles and preserves behavior.

I just tested and that is the case.

Reply:
I guess it does compile and is "correct" but it isn't what we would want with aliasing:

```
        expexted_output = """import tempfile as _tempfile

_tempfile.mkstemp()
var = "hello"
"""
```
instead the result is 
```
import tempfile

tempfile.mkstemp()
var = "hello"
```

which is fine  but the point of the unit test is to show we would correctly handle aliasing which we don't. 
I think it's interesting though that it does return legitimate code. So maybe we should become less strict with aliasing and accept that the user's aliasing will be overriden?

Reply:
Ideally we want to respect aliasing. We are inviting possible errors otherwise, (e.g. shadowing names). I believe those would be a bit rare though.